x-pack/plugins/apm/common/agent_configuration/setting_definitions/java_settings.ts

```diff
@@ -176,7 +184,16 @@ export const javaSettings: RawSettingDefinition[] = [
           'The frequency at which stack traces are gathered within a profiling session. The lower you set it, the more accurate the durations will be. This comes at the expense of higher overhead and more spans for potentially irrelevant operations. The minimal duration of a profiling-inferred span is the same as the value of this setting.'
       }
     ),
-    includeAgents: ['java']
+    includeAgents: ['java'],
+    validation: getDurationRt([
+      // must be between 1ms(1ms - 1000ms) and 1s
+      { min: 1, max: 1, unit: 's' },
+      { min: 1, max: 1000, unit: 'ms' }
+    ]),
```

Comment:
is there a reason why we would need to ever define multiple boundaries? in this case, both boundaries are the same.

Reply:
I made some changes... take a look b3e9915